tests/api-testing/tests/test_foldersnew.py

```diff
@@ -0,0 +1,142 @@
+    resp_create_new_dashboards = session.post(
+        f"{url}api/v2/{org_id}/folders/dashboards", json=payload)
+
+    print(resp_create_new_dashboards.content)
+
+    folder_id = resp_create_new_dashboards.json()["folderId"]
+
+    assert (
+        resp_create_new_dashboards.status_code == 200
+    ), f"Expected 200 dashboards, but got {resp_create_new_dashboards.status_code} {resp_create_new_dashboards.content}"
+    
+    payload = {
+        "description": "folderupdated",
+        "folderId": folder_id,
+        "name": "pytest-automation",
+    }
+    folder_id = resp_create_new_dashboards.json()["folderId"]
+    # Update folder
+    resp_update_new_dashboards = session.put(
+        f"{base_url}api/v2/{org_id}/folders/dashboards/{folder_id}", json=payload
+    )
+    assert (
+        resp_update_new_dashboards.status_code == 200
+    ), f"updating this dashboard folder, but got {resp_update_new_dashboards.status_code} {resp_update_new_dashboards.content}"
+    print(resp_update_new_dashboards.content)
+    # Delete Dashboard Folder
+    resp_delete_new_dashboards = session.delete(
+        f"{base_url}api/v2/{org_id}/folders/dashboards/{folder_id}")
+    assert (
+        resp_delete_new_dashboards.status_code == 200
+    ), f"Deleting this Dashboards folder, but got {resp_delete_new_dashboards.status_code} {resp_delete_new_dashboards.content}"
+
+def retrieve_new_alert(session, base_url, org_id, folder_id):
+    resp_get_alert = session.get(
+        f"{base_url}api/v2/{org_id}/folders/alerts/{folder_id}")
+    print(resp_get_alert.content)
+    assert (
+        resp_get_alert.status_code == 200
+    ), f"Retrieve this folder alert: Expected 200, but got {resp_get_alert.status_code} {resp_get_alert.content}"
+
+
+def test_new_alertsall(create_session, base_url):
+    """Running an E2E test for all new alerts."""
+
+    session = create_session
+    org_id = "default"
+
+    resp_new_all_alerts = session.get(f"{base_url}api/v2/{org_id}/folders/alerts")
+
+    print(resp_new_all_alerts.content)
+    assert (
+        resp_new_all_alerts.status_code == 200
+    ), f"Retrieve all functions list 200, but got {resp_new_all_alerts.status_code} {resp_new_all_alerts.content}"
+
+def test_new_create_alerts_update_delete(create_session, base_url):
+    """Running an E2E test for create and delete alerts folder."""
+
+    session = create_session
+    # Create a alerts folder
+    org_id = "default"
+    url = base_url
+    payload = {"description": "newfoldernvp", "name": "pytest-automationtests"}
+    resp_create_new_alerts = session.post(
+        f"{url}api/v2/{org_id}/folders/alerts", json=payload)
+
+    print(resp_create_new_alerts.content)
+
+    folder_id = resp_create_new_alerts.json()["folderId"]
+
+    assert (
+        resp_create_new_alerts.status_code == 200
+    ), f"Expected 200 alerts, but got {resp_create_new_alerts.status_code} {resp_create_new_alerts.content}"
```

Comment:
This test is creating the "alerts folder" not the "alert". I'd recommend updating to update the logs to reflect that in order to avoid confusion.